docs_src/OVMS/runObjectDetectionPipelineWithNewModel.md

```diff
@@ -0,0 +1,33 @@
+# Run Object Detection Pipeline with New Model
+For running object detection pipeline, we have defaulted the model to person_vehicle_bike_detection_2000. You can use different model to run object detection. Here are the steps:
+
+1. Add new section to config file for model server
+2. Download new model
+3. Add new model label file
+4. Add architecture type of new model as input to pipeline
+5. rebuild containers
+
+
+## Add New Section to Config File for Model Server
+Here is the config file location: `configs/opencv-ovms/models/2022/config.json`, edit the file and append following configuration section template
```

Comment:
Users should be encouraged to update the https://github.com/intel-retail/automated-self-checkout/blob/main/configs/opencv-ovms/models/2022/config_template.json.

This way the "target_device": "{target_device}"}, will be set based on the $DEVICE environment variable.